doc/04-Getting-started.md

```diff
@@ -28,6 +28,67 @@ this is an advanced task you should not tackle at the early beginning.
 
 Take some time to really understand how to work with Icinga Director first.
 
+First steps on the Frontend
+====================================
+Create a host
+------------------
+
+First we create a `host template` which we will name `director-generic-host`. The default check we will use for a Host is `hostalive`.
+
+Click on the interface on Icinga Director > Host Objects > Add Host and `add a template`. For details have a look at the screenshots below:
+
+![add host section](screenshot/director/04-getting-started/01-add_a_host.png)
+
```

Comment:
Screenshot should be made with active object type field, that would highlight the essential component. Default host should NOT be an agent, please reset that field and/or collapse the whole zone/cluster section. Also, it is a VERY bad idea to define your default host `volatile`.